### PR TITLE
Updated to the replacement of Device.Addressable with Device Protocols

### LINKED ISSUE #33: Update due to Device changes and Schedule* deprecation
The example configuration file needs to be updated to the replacement of `Device.Addressable` with `Device.Protocols`, and the deprecation of Schedule* objects by device-sdk-go.

The same `Device` **configuration.toml** and example driver changes that were made to device-sdk-go in commit [fc71221](https://github.com/edgexfoundry/device-sdk-go/commit/fc712212962e7a52da8ff0b7d6a8734cc5af8bdd) need to be applied. 

Likewise the Schedule* **configuration.toml** changes (see device-sdk-go commit [2db9552](https://github.com/edgexfoundry/device-sdk-go/commit/2db9552350484545cc2f0750a9a33b6f572fc05c) also need to be applied.

### PR DESCRIPTION
**Pull request by weichou1229**
Fix #33 
Updated to the replacement of Device.Addressable with Device Protocols, and the deprecation of Schedule* objects by device-sdk-go.

Signed-off-by: Bruce <weichou1229@gmail.com>

### CODE REVIEW

File: internal/driver/config.go [hunk 1]
```diff
@@ -30,6 +30,27 @@ type SubscribeInfo struct {
...
+	Schema   string
+	Host     string
+	Port     string
+	User     string
+	Password string
+	ClientId string
+	Topic    string
+}
+
+const (
```
Comment (cloudxxx8) [inline internal/driver/config.go]: move all const to a separated file, called "protocolpropertykey.go"

Comment (weichou1229) [reply internal/driver/config.go]: Modified.

**Comment by cloudxxx8** [comment]
Since the SDK API is changed to use ProtocolProperties, could you please rework on this?
thanks

**Comment by weichou1229** [comment]
I have modified the func according to the change.

**Comment by cloudxxx8** [comment]
recheck

**Pull request merged**